CHANGELOG.md

```diff
@@ -37,14 +37,14 @@ maintainability of worker queries, and improve DB performance. ([PR](https://git
   [our worker/controller compatiblity policy](https://developer.hashicorp.com/boundary/docs/enterprise/supported-versions#control-plane-and-worker-compatibility).
 * Add concurrency limit on the password hashing of all password auth methods.
   ([PR](https://github.com/hashicorp/boundary-plugin-aws/pull/5437)).
-
+  
```

Comment:
Looks like extra whitespace here got added

Reply:
Yea I pulled the changelog on main and confirmed it exists there too, so I left it. Do you suggest I remove it?

Reply:
Yup it might be your editor adding it